tests/software/bcftools/tabix/test.yml

```diff
@@ -0,0 +1,8 @@
+- name: Run bcftools tabix test workflow
```

Comment:
```suggestion
- name: bcftools tabix
```

Reply:
Can you rename all of these to just `software tool` please?

Reply:
Solved here https://github.com/nf-core/modules/pull/134/commits/f8882de45613b8afe068d9a8d6df86c35cc95854